base.yaml

```
@@ -3,7 +3,7 @@ external_components:
   - source:
       type: git
       url: https://github.com/ratgdo/esphome-ratgdo
-      ref: main
+      ref: ram
```

```suggestion
      ref: main
```

Revert before merging